From: Thomas Bogendoerfer <tbogendoerfer@suse.de>
Subject: kabi i40e ignore include
Patch-mainline: never, kabi
References: jsc#SLE-4797

Upstream commit f3fef2b6e1cc ("i40e: Remove umem from VSI")
includes a new header file, which confuses kABI checker. Hide
this #include from kABI checker.

Signed-off-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/intel/i40e/i40e.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/net/ethernet/intel/i40e/i40e.h
+++ b/drivers/net/ethernet/intel/i40e/i40e.h
@@ -57,7 +57,9 @@
 #include <net/pkt_cls.h>
 #include <net/tc_act/tc_gact.h>
 #include <net/tc_act/tc_mirred.h>
+#ifndef __GENKSYMS__
 #include <net/xdp_sock.h>
+#endif
 #include "i40e_type.h"
 #include "i40e_prototype.h"
 #include "i40e_client.h"
